package proto_ff;

import "yd_fieldoptions.proto";


message E_MeditationMeditationAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_MeditationMeditationActiveattributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_MeditationMeditation
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "打坐ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "打坐名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "外观类型"];
	optional int32 m_quality = 4[(yd_fieldoptions.field_cname) = "打坐品质"];
	optional string m_professionlimit = 5[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 64];
	optional int64 m_activationitem = 6[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 m_activationnum = 7[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 m_starid = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string m_starnum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 256];
	optional int32 m_starup = 10[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 m_starber = 11[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated E_MeditationMeditationAttributeDesc m_attribute = 12[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 8];
	repeated E_MeditationMeditationActiveattributeDesc m_activeattribute = 13[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 8];
}


message Sheet_MeditationMeditation
{
	repeated E_MeditationMeditation E_MeditationMeditation_List = 1[(yd_fieldoptions.field_arysize)=16];
}
